diagnostic.md

```diff
@@ -22,9 +21,7 @@ at the `OL` DOM node).
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$(li).on
```

Comment:
actually, it would be `$('li');`. Also, TWO ways! Another would be `$('ol > li');`